pressbooks-multi-institution.php

```diff
@@ -33,6 +36,17 @@
 }
 
 register_activation_hook(__FILE__, [Migration::class, 'migrate']);
-register_deactivation_hook(__FILE__, [Migration::class, 'rollback']);
+register_activation_hook(__FILE__, function () {
+    $managerIds = InstitutionUser::query()->where(['manager' => true])->pluck('user_id')->toArray();
```

Comment:
There's a `scopeManagers` in the `InstitutionUser` class, meaning you could use it like this: 

```suggestion
    $managerIds = InstitutionUser::query()->managers()->pluck('user_id')->toArray();
```